Model/Product/Sku/Collection.php

```diff
@@ -45,6 +45,7 @@
 use Nosto\Tagging\Model\Product\Repository as NostoProductRepository;
 use Nosto\Tagging\Model\Product\Sku\Builder as NostoSkuBuilder;
 use Nosto\Types\Product\SkuInterface;
+use Magento\ConfigurableProduct\Model\ResourceModel\Product\Type\Configurable\Attribute\Collection as ConfigurableAttributeCollection; // @codingStandardsIgnoreLine
```

Comment:
We can also break the line here and remove the annotation
```suggestion
use Magento\ConfigurableProduct\Model\ResourceModel\Product\Type\Configurable\Attribute\Collection 
    as ConfigurableAttributeCollection;
```